### PR TITLE
Nested pools release

### PR DESCRIPTION
**Pull request by johngrantuk**

### CODE REVIEW

File: examples/removeLiquidity/removeLiquidityNested.V3.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
+/**
+ * Example showing how to remove liquidity from a V3 nested pool.
+ * V3 nested removes are always proportional.
+ * User must approve composite router to spend BPT token - See tests in test/v3/removeLiquidityNested for full flow.
+ * (Runs against a local Anvil fork)
+ * Note - remove transaction is not fully run in this example because we cannot use slot method to artificially give test account BPT balance.
```
Comment (MattPereira) [inline examples/removeLiquidity/removeLiquidityNested.V3.ts:6]: Should be solve-able when we get around to the major refactor of examples

File: examples/addLiquidity/addLiquidityNested.V3.ts
Comment (MattPereira) [inline examples/addLiquidity/addLiquidityNested.V3.ts:1]: Hell of a lot easier to follow than `makeForkTx` :clap:

**Review by MattPereira — APPROVED** [review]
Well done ser! :beers:

File: examples/removeLiquidity/removeLiquidityNested.V3.ts (hunk 1 above)
Comment (johngrantuk) [reply examples/removeLiquidity/removeLiquidityNested.V3.ts:6]: Yeah, exactly. Didn't want to get too deep into that at this stage.

**Pull request merged**